### PR TITLE
[Remove unused reference] LPS-165338 | updated #2085

### PR DESCRIPTION
**Pull request by dantewang**
@JanisZhang Compared to #2085, I

- I removed https://github.com/liferay-core-infra/liferay-portal/pull/2085/commits/692339fd27fb4723d7e2e1091b36a4ccf01c3eba, because on the call chain of `DDMFormLayoutFactory.create` there's `PortalUtil` usage.
- I modified https://github.com/liferay-core-infra/liferay-portal/pull/2085/commits/e69b46d201595b04d61cba36660faf573952566a to make `_jsonFactory` used instead, because `DDMFormFieldValueExpressionParameterAccessor` uses `JSONFactoryUtil`.

cc @W2jx @slnn

Basically I focus on static method called, and classes created, by the modified component class.

You can modifiy IntelliJ's Editor configuration, in "配色方案" - "Java" - "方法 / 静态方法" and "方法 / 构造函数调用". Personally I modified the static method to have the same color as static field and constant, and made constructor invocations bold.

### CODE REVIEW

**Comment by dantewang** [comment]
image
image

Looks like

image

**Comment by Kvale1733** [comment]
ci:test:data-engine

**Comment by Kvale1733** [comment]
@dantewang No regressions found.

**Comment by dantewang** [comment]
ci:reevaluate:1387401_4388

**Comment by dantewang** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/125021

cc @shuyangzhou @tinatian
